<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SyncResponseWindow</class>
 <widget class="QWidget" name="SyncResponseWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>376</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">SyncResponseWindow{
	background-color: #F7F4F8;
}
QLabel{
	color: black;
}</string>
  </property>
  <widget class="QLabel" name="lblTitle">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>20</y>
     <width>301</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">SyncResponseWindow{
	background-color: #F7F4F8;
}
QLabel{
	color: black;
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="lblT">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>80</y>
     <width>81</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Event:</string>
   </property>
  </widget>
  <widget class="QLabel" name="lblEventTitle">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>80</y>
     <width>361</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="lblD">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>130</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Duration:</string>
   </property>
  </widget>
  <widget class="QLabel" name="lblDuration">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>120</y>
     <width>231</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QWidget{
	backgroun-color: #F7F4F8;
}
QLabel{
	color: black;
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="btnYes">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>240</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Yes</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnNo">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>244</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>No</string>
   </property>
  </widget>
  <widget class="QLabel" name="lblQuestion">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>190</y>
     <width>141</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Add to calendar?</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
